GovUkDesignSystemComponents/SubComponents/SummaryListRow.cshtml

```diff
@@ -0,0 +1,15 @@
+﻿@model GovUkDesignSystem.GovUkDesignSystemComponents.SummaryListRowViewModel
+
+@{
+    string classNames =
```

Comment:
Please inline classNames for consistency, here and below

Reply:
Done, thanks